### PR TITLE
[ENHANCEMENT] - Allow shortcode aliases

### PR DESCRIPTION
**Pull request by loranger**
This PR provides a workaround for #42 by implementing a simple aliases sytem

### CODE REVIEW

File: src/LitEmoji.php [hunk 1]
```diff
@@ -183,6 +184,15 @@ public static function config(string $property, $value): void
                 self::$entityCodepoints = [];
                 self::$excludedShortcodes = [];
                 break;
+            case 'includeShortcodes':
```
Comment (joshmcrae) [inline src/LitEmoji.php:187]: Call the configuration option `aliasShortcodes`.

File: tests/LitEmojiTest.php [hunk 2]
```diff
@@ -44,8 +44,8 @@ public function testHtmlToUnicode()
 
     public function testUnicodeToShortcodeTiming()
     {
-        $text = LitEmoji::encodeShortcode(file_get_contents(__DIR__ . '/UnicodeIpsum'));
-        $this->assertEquals(file_get_contents(__DIR__ . '/ShortcodeIpsum'), $text);
+        $text = LitEmoji::encodeShortcode(file_get_contents(__DIR__.'/UnicodeIpsum'));
```
Comment (joshmcrae) [inline tests/LitEmojiTest.php:47]: Could you please revert the removal of whitespace around `.`. Perhaps this was an automatic change based on your editor settings, but it's our code style preference that this remains as it was.

**Review by joshmcrae — CHANGES_REQUESTED** [review]
Looks great! I've requested a few changes below based on code style and naming.

Once we merge this, I'll also merge #43 and make sure they can be used side-by-side.

File: src/LitEmoji.php [hunk 3]
```diff
@@ -9,6 +9,7 @@ class LitEmoji
     private static array $shortcodeEntities = [];
     private static array $entityCodepoints = [];
     private static array $excludedShortcodes = [];
+    private static array $aliasesShortcodes = [];
```
[the remark this answers is not in the document]
Comment (loranger) [reply src/LitEmoji.php]: Done !

File: src/LitEmoji.php (hunk 1 above)
Comment (loranger) [reply src/LitEmoji.php]: Done and documentation is up-to-date too

File: tests/LitEmojiTest.php (hunk 2 above)
Comment (loranger) [reply tests/LitEmojiTest.php]: Definitely a phpcsfixer rule, I did revert those changes

**Review by joshmcrae — APPROVED** [review]
Thanks!

I'll make a couple of changes while merging #43 and then do a single release.

**Comment by joshmcrae** [comment]
Thanks for your contribution @loranger.

Released in [5.1.0](https://github.com/elvanto/litemoji/releases/tag/5.1.0).

**Pull request merged**